packages/core/src/components/DataTable/index.tsx

```diff
@@ -447,6 +448,9 @@ export default withStyles(
       borderBottom: ui.border,
       cursor: 'pointer',
     },
+    column_header_bold: {
+      borderBottomWidth: ui.borderWidthThick,
```

Comment:
Personally I think this should be the default, and would help distinguish the table header from the rest of the table rows.